generated/nidcpower/attributes.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+# This file was generated
+
+
+class Attribute(object):
+    '''Base class for all typed attributes.'''
+
+    def __init__(self, attribute_id):
+        self._attribute_id = attribute_id
+
+
+class AttributeViInt32(Attribute):
```

Comment:
Shouldn't there also be an AttributeViInt64?

Reply:
Yeah. I need to add the class and also an example of it to NI-FAKE + Unit test.

Reply:
Fixed by Marcos